docs/reference/camera.md

```diff
@@ -1188,6 +1224,25 @@ The `wb_camera_recognition_get_sampling_period` function returns the period give
 
 The `wb_camera_recognition_get_number_of_objects` and `wb_camera_recognition_get_objects` functions allow the user to get the current number of recognized objects and the objects array.
 
+*camera recognition segmentation functions*
+
+If a [Recognition](recognition.md) node is present in the `recognition` field and the [Recognition](recognition.md).`segmentation` field is set, the camera generates a segmentation ground truth image corresponding to the camera image.
+The segmented image is generated at the same sampling period as the recognition objects.
+For the segmentation to work it is necessary to enable the recognition, but it is not necessary to enable the camera.
+
+The `wb_camera_recognition_has_segmentation` function can be used to check if the [Recognition](recognition.md).`segmentation` field is set to TRUE.
+If the [Recognition](recognition.md) node is not defined, the function returns FALSE.
+
+The `wb_camera_recognition_enable_segmentation` and `wb_camera_recognition_disable_segmentation` functions toggle the generation of the segmented image.
+Note that the generation of the segmented image can only be enabled if the recognition functionality is enabled (see [`wb_camera_has_recognition`](camera.md#wb_camera_has_recognition) and [`wb_camera_recognition_enable`](camera.md#wb_camera_recognition_enable)).
```

Comment:
This is not true.
The user has to explicitly enable the segmentation using the `wb_camera_recognition_enable_segmentation` after enabling the recognition.

To me it seems better to let the user have to enable explicitly the segmentation feature rather than force him to explicitly disable it when he doesn't want it.

Reply:
I agree it is better this way, so we should probably state it clearly in the documentation as well, as I was unsure of the behavior from the description, like:
```
Even if the [Recognition](recognition.md).`segmentation` field is TRUE, the generation of segmented images is not automatically enabled when calling the [`wb_camera_recognition_enable`](camera.md#wb_camera_recognition_enable) function, so it is necessary to call the `wb_camera_recognition_enable_segmentation` function to enable the generation of segmented images.
```